solidity/migrations/external-contracts.js

```diff
@@ -1,12 +1,12 @@
 // Addresses of externally deployed smart contracts
-const RegistryAddress = '0xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA'
-const TokenStakingAddress = '0xBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB'
-const RandomBeaconAddress = '0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC'
-const TBTCSystemAddress = '0xDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD'
+const RegistryAddress = "0x9aF7d664E6d7DB318c6BdE54ED77A7D99361F539"
+const TokenStakingAddress = "0x602c5f7d11eb786E14cDfBafe34E70c0b6717a8c"
+const RandomBeaconAddress = "0xB901EDDE70EcB59c1243f404a238fd97B78f1F54"
+const TBTCSystemAddress = "0x065c3Ab1D2cb67c70dAC17Cf3BD4B8A165ffceFa"
```

Comment:
Is this on purpose? <_<

Reply:
Oh boy, no! It's fixed in https://github.com/keep-network/keep-ecdsa/pull/338/commits/43bfb27ab472dd21a4725ea6f27503eb09496eba.
Thanks for noticing that!